src/Properties.ts

```diff
@@ -0,0 +1,28 @@
+import { Logger } from '@hmcts/nodejs-logging'
+import * as fs from 'fs'
+
+const logger = Logger.getLogger('applicationRunner')
```

Comment:
doesn't look like the right name for the logger

Reply:
sure Ill change it to log